modular_skyrat/modules/modular_implants/code/nifs.dm

```diff
@@ -0,0 +1,511 @@
+/// This is the original NIF that other NIFs are based on.
+/obj/item/organ/internal/cyberimp/brain/nif
+	name = "Nanite Implant Framework"
+	desc = "a brain implant that infuses the user with nanites" //Coder-lore. Change this later
```

Comment:
Would be nice to see this changed, also let's make it actually grammatically-sound at the very least.
```suggestion
	desc = "A brain implant that infuses the user with nanites." //Coder-lore. Change this later
```

Reply:
Fair, but there is no way for players to actually be able to obtain this model of NIF ingame, aside from having an admin spawn it in.